.appveyor.yml

```diff
@@ -9,6 +9,13 @@ branches:
 
 clone_depth: 64
 
+skip_commits:
+  files:
+    - docs/*
+    - NEWS
+    - UPGRADING
+    - UPGRADING.INTERNALS
```

Comment:
Can we include:
```suggestion
    - UPGRADING.INTERNALS
    - *.md
```

Reply:
i took the same "excludes" as we have for azure, see comment in https://github.com/php/php-src/pull/4352